### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by hzsweers**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,8 +6,12 @@ jdk:
 
 android:
   components:
-    - build-tools-21.1.2
-    - android-21
+    - build-tools-22.0.1
+    - android-22
+    - extra-android-support
+    - extra-google-google_play_services
```
Comment (JakeWharton) [inline .travis.yml]: This and the item above it is not needed.

Comment (hzsweers) [inline .travis.yml]: Ah, I see. I played with different combinations of this for awhile trying to get the travis build to work, and wasn't 100% sure what was needed. Sorry!

Comment (JakeWharton) [inline .travis.yml]: Not a big deal. Thanks for the work!

Comment (hzsweers) [inline .travis.yml]: :+1:

**Pull request merged**